sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -50,54 +51,65 @@ import Foundation
     var kind: IdentifierKind { get }
 }
 
+internal enum Prefix {
+    public static let PhoneNumber = "4:"
```

Comment:
don't forget to update the other classes with the hardcoded constants...such as `public class PhoneNumberIdentifier`

Reply:
it's already updated